src/phasorpy/phasor.py

```diff
@@ -67,6 +67,10 @@
 
   - :py:func:`phasor_filter`
 
+- threshold phasor coordinates based on mean intensity:
```

Comment:
In a way, it is a filter function. Move it in one category with `phasor_filter`?